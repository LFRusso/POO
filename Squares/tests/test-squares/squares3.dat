Square0 0.332788 -0.759821 1.52425
Square1 1.15565 2.46332 0.836315
Square2 0.101182 0.0455936 2.73085
Square3 1.60998 2.1599 4.81254
Square4 -1.54361 0.215639 0.321843
Square5 2.90044 -2.80214 4.84557
Square6 1.71624 1.69707 2.01216
