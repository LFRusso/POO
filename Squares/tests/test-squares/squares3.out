Square0 intercepts Square2 with area 1.09568
Square1 intercepts Square2 with area 0.26187
Square1 intercepts Square3 with area 0.31946
Square1 intercepts Square6 with area 0.230593
Square2 intercepts Square3 with area 0.753448
Square2 intercepts Square6 with area 1.20436
Square3 intercepts Square6 with area 3.1175
Square5 intercepts Square6 with area 0.286772
